pupil-spa/src/app/services/submission/submission.service.ts

```diff
@@ -9,6 +9,25 @@ export class SubmissionService {
   constructor(private http: Http, private storageService: StorageService) {
   }
 
+  async submitCheckStartData() {
+    const headers = new Headers();
+    headers.append('Content-Type', 'application/json');
+    const requestArgs = new RequestOptions({headers: headers});
+    const { checkCode } = this.storageService.getItem('pupil');
+    const accessToken = this.storageService.getItem('access_token');
+    await this.http.post(`${environment.apiURL}/api/check-started`,
+      { checkCode, accessToken },
+      requestArgs)
+      .toPromise()
+      .then((response) => {
+          if (response.status !== 201) {
+            return new Error('Submit Error:' + response.status + ':' + response.statusText);
+          }
+        },
+        (err) => { throw new Error(err); }
+      ).catch((err) => { throw new Error(err); });
```

Comment:
Will this api call make any retries if the server is busy? E.g. 504, or 504?  It probably should back off and retry.